### PR TITLE
Update tslint rules with latest from nx

### PR DESCRIPTION
**Pull request by izifortune**

### CODE REVIEW

File: packages/typescript/angular.js [hunk 1]
```diff
@@ -6,21 +6,36 @@ module.exports = {
...
+      true,
+      "element",
+      "app",
+      "kebab-case"
+    ],
     "use-input-property-decorator": true,
-    "use-life-cycle-interface": true,
     "use-output-property-decorator": true,
+    "use-host-property-decorator": true,
+    "no-input-rename": true,
```
Comment (margaris23) [inline packages/typescript/angular.js]: there are cases we deliberately rename inputs due to conflicting names with inner variables and we also want to keep a more clear name for input. in my opinion this rule is strict

File: packages/typescript/angular.js [hunk 2]
```diff
@@ -6,21 +6,36 @@ module.exports = {
...
+    "no-output-rename": true,
+    "use-life-cycle-interface": true,
     "use-pipe-transform-interface": true,
+    "component-class-suffix": true,
+    "directive-class-suffix": true,
+    "no-access-missing-member": true,
+    "templates-use-public": true,
+    "invoke-injectable": true,
```
Comment (margaris23) [inline packages/typescript/angular.js]: I cannot find invoke-injectable rule. what does it do?

File: packages/typescript/index.js [hunk 3]
```diff
@@ -55,6 +55,86 @@ module.exports = {
...
+    "import-spacing": true,
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "interface-over-type-literal": true,
+    "label-position": true,
+    "max-line-length": [
+      true,
+      140
```
Comment (margaris23) [inline packages/typescript/index.js]: I believe 140 is the default. rooms uses 100. we have to decide about that.

Comment (AIOrc) [reply packages/typescript/index.js]: We are using 100 as well.

File: packages/typescript/index.js [hunk 4]
```diff
@@ -55,6 +55,86 @@ module.exports = {
...
+    "no-non-null-assertion": true,
+    "no-shadowed-variable": true,
+    "no-string-literal": false,
+    "no-string-throw": true,
+    "no-switch-case-fall-through": true,
+    "no-unnecessary-initializer": true,
+    "no-unused-expression": true,
+    "no-use-before-declare": true,
+    "no-var-keyword": true,
+    "object-literal-sort-keys": false,
```
Comment (AIOrc) [inline packages/typescript/index.js]: Is it too much of a hassle to turn this one on?

Comment (izifortune) [reply packages/typescript/index.js]: Linters will fix it automatically

File: packages/typescript/angular.js (hunk 2 above)
Comment (izifortune) [reply packages/typescript/angular.js]: deprecated i think from codelyzer i will remove it

**Pull request closed without merging**